SetupDataPkg/Tools/ConfigEditor.py

```diff
@@ -897,64 +986,118 @@ def save_to_delta(self):
     def save_full_to_delta(self):
         self.save_delta_file(True)
 
-    def save_to_bin(self):
+    def save_to_raw_bin(self):
+        self.save_to_bin(False)
+
+    def save_to_var_list_bin(self):
+        self.save_to_bin(True)
+
+    def save_to_bin(self, variable_list_format):
```

Comment:
Similar to the other comment, consider rename the input argument to make it more self-explanatory.